examples/plot_simulate_evoked.py

```diff
@@ -1,15 +1,20 @@
 """
-===============
-Simulate dipole
-===============
+====================================
+Simulate dipole using new drives-API
```

Comment:
I don't want to keep a distinction between old and new API. There should be only one recommended way and that is in my opinion the new API.

Reply:
Oops, missed that.